react/react1/week1/to-do-app/src/ToDo.css

```diff
@@ -0,0 +1,31 @@
+body {
```

Comment:
also, you can put ToDo.css in a separate /components folder. As I understand, ideally each bigger component should have its own folder and own css - e.g. /ToDoComponent/ToDo.jsx & /ToDoComponent/ToDo.css - then it is more organized. Not needed now - it's not too big :)